picker/picker.go

```diff
@@ -165,7 +165,7 @@ const (
 	// See
 	// https://en.wikipedia.org/wiki/Exponential_smoothing#Basic_exponential_smoothing
 	// for details.
-	remoteWeightSmoothingFactor = 0.7
+	remoteWeightSmoothingFactor = 0.999
```

Comment:
Actually, we want this around 0.5 or it will have no effect. Remove this line and we'll merge and then I'll submit a follow up.